config.ru

```diff
@@ -8,5 +8,6 @@ Sidekiq.configure_client do |config|
   config.redis = {url: ENV['REDIS_URL']}
```

Comment:
We may want to cascade this as `SIDEKIQ_REDIS_URL` then to `REDIS_URL`, for a scenario where we have sidekiq running on a separate redis to cache/timelines.